entries/t/trustage.com.json

```diff
@@ -0,0 +1,12 @@
+{
+    "Trustage": {
+      "domain": "trustage.com",
+      "url": "https://www.trustage.com/",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```

Reply:
This still needs fixing ^